frontends/web/src/routes/account/send/send.tsx

```diff
@@ -503,12 +514,14 @@ class Send extends Component<Props, State> {
             activeScanQR: true,
             videoLoading: true,
         }, () => {
-            this.qrCodeReader
+            this.qrCodeReader && this.qrCodeReader
                 .decodeFromInputVideoDevice(undefined, 'video')
                 .then(result => {
                     this.setState({ activeScanQR: false });
                     this.parseQRResult(result.getText());
-                    this.qrCodeReader.reset(); // release camera
+                    if (this.qrCodeReader) {
```

Comment:
Didn't you already check this a couple lines above with `this.qrCodeReader && this.qrCodeReader.decode...`?

Reply:
Yes and if it was up for me that would be perfectly fine, because I know we are not overwriting `this.qrCodeReader` anywhere else. But Typescript insisted and I didn't want to shut him up, the issue is `.then(` is async and in the meantime something could in theory have set `this.qrCodeReader` to undefined again. I am not sure what to do in these situations and just do whatevery Typescript tells me to do 🙈  

![Screen Shot 2021-03-05 at 2 28 04 PM](https://user-images.githubusercontent.com/546900/110121757-1507f280-7dbf-11eb-833b-33e0a769aedc.png)